### PR TITLE
Apply pagination strip

### PR DESCRIPTION
**Pull request by mikeminutillo**
https://github.com/Particular/ServicePulse/pull/1665 added a pagination strip re-usable component. This PR applies the component everywhere in the application where pagination has been added to provide a consistent user experience.

### CODE REVIEW

**Review by johnsimons — APPROVED** [review]
very nice to see soo much code being deleted

**Comment by johnsimons** [comment]
@mikeminutillo the only comment is it seems we are repeating `const per page = 50;` in all the pages.
I'm wondering if that should be the component default.

**Comment by mikeminutillo** [comment]
> @mikeminutillo the only comment is it seems we are repeating `const per page = 50;` in all the pages. I'm wondering if that should be the component default.

The component can't have a default unless the call to service control also passes the same value (i.e. the SC page default is 50). If either of those change we will get an inconsistent UI. I opted to make it explicit and always pass it to SC to guarantee consistency. It could maybe be extracted as an app-wide constant that gets imported in where it is used.

File: src/ServicePulse.Host/vue/src/components/failedmessages/AllFailedMessages.vue [hunk 1]
```diff
@@ -45,6 +46,10 @@ const sortOptions = [
   },
 ];
 
+watch(pageNumber, () => {
```
Comment (PhilBastian) [inline src/ServicePulse.Host/vue/src/components/failedmessages/AllFailedMessages.vue:49]: remove the curly brackets so this gets single lined like the file above

**Review by PhilBastian — APPROVED** [review]

**Pull request merged**